pep-0001.txt

```diff
@@ -148,10 +148,26 @@ The standard PEP workflow is:
 * You, the PEP author, fork the `PEP repository`_, and create a file named
   ``pep-9999.rst`` that contains your new PEP.  Use "9999" as your draft PEP
   number.
+
 * Push this to your GitHub fork and submit a pull request.
-* The PEP editors review your PR for structure, formatting, and other errors.
+
+* The PEP editors review your PR for structure, formatting, and other
+  errors.  Approval criteria are:
+
+  * It sound and complete.  The ideas must make technical sense.  The
+    editors do not consider whether they seem likely to be accepted.
+  * The title accurately describes the content.
+  * The PEP's language (spelling, grammar, sentence structure, etc.)
+    and markup (for reST PEPs) are correct, and code style (examples
+    should match PEP 8 & PEP 7) is good.  The PEP editors will edit
+    language and code style, but they can't approve it if they can't
+    parse it.  Do your best.
+
+  If the PEP isn't ready for approval, an editor will send it back to
+  the author for revision, with specific instructions.
+
 * Once approved, they will assign your PEP a number, and label it as Standards
-  Track, Informational, or Process, and give it the status "Draft"
+  Track, Informational, or Process, and give it the status "Draft".
```

Comment:
Hm, typically this is done by the PEP author since every submission now starts as a PR. (Of course the PEP editors will review that the label is the right one.)

Reply:
This is a direct quote from "PEP Editors' Responsibilities & Workflow".  I didn't feel comfortable changing it.

I will submit a revised PR with this change in a separate commit (later, it's bedtime here).